rust/tools/authorization-logic/.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+rustflags = ["-C", "target-feature=+crt-static"]
```

Comment:
It sounds like it might be best not to force the build to always be static or target a particular architecture. So now there is just a shell script to build the binary instead. Does that work?